target/linux/ramips/dts/mt7621_humax_e10.dts

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "humax,e10", "mediatek,mt7621-soc";
+	model = "HUMAX E10";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &wan;
+	};
+	
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_usb_power {
```

Comment:
The convention is dash for node names, underscore for label names.

Reply:
Done!